BackEnd/config/db.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose')
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
> As early as possible in your application, import and configure dotenv

Ref - https://www.npmjs.com/package/dotenv

Please move this piece of code to the top of `server.js`